Loop/View Controllers/BolusViewController.swift

```diff
@@ -572,8 +572,23 @@ final class BolusViewController: ChartsTableViewController, IdentifiableClass, U
         guard let amount = enteredBolusAmount else {
             return nil
         }
+        
+        // ANNA TODO: improve ~style~
+        var deliveredUnits: Double
+        
+        if let model = enteredBolusInsulinModel as? ExponentialInsulinModelPreset {
+            switch model {
+            case .afrezza:
+                // The manufacturer has said that 4 units of Afrezza is roughly the same as 3 units of rapid-acting
+                deliveredUnits = 0.75 * amount
```

Comment:
I believe this varies greatly on a per-person basis (much more than for example deviations from typical/pre-defined rapid-acting SC curve). If this model is to be included, perhaps a configurable option for potency should be included (same as the Walsh model is configurable). Just my 2 cents.

Reply:
Yes, this could vary. I think at this point, it makes the most sense to move all of the Afrezza work to a separate branch to keep the discussion focused on the underlying architecture changes. Thanks for the feedback!